web/settings/modulconfiglp.php

```diff
@@ -749,6 +749,11 @@ function visibility_twcmanagerlp1_connection() {
 										<option <?php if($socmodulold == "soc_zerong") echo "selected" ?> value="soc_zerong">Zero NG</option>
 									</optgroup>
 								</select>
+								<div id="socmvwidinfo" class="mt-1 alert alert-info hide">
```

Comment:
Bitte den Block in das schon vorhandene DIV `#socmvwid` einfügen. Separates Ein-/Ausblenden ist dann nicht mehr erforderlich und der Code ist nicht verstreut.